settings/development.py

```diff
@@ -10,6 +10,8 @@
 WSGI_APPLICATION = 'wsgi.heroku.application'
 
 STATICFILES_STORAGE = 'storage.NonPackagingS3PipelineCachedStorage'
+DEFAULT_FILE_STORAGE = 'storage.MediaStorage'
```

Comment:
This isn't needed in this branch for hosting static files, esp since widget users aren't uploading logos.